### DISCUSSION TITLE
Ability to have Modrinth email mod author when a file is uploaded under their account

### DISCUSSION THREAD

**Discussion by TelepathicGrunt** [modrinth/.github#110] [Feature requests]
This will allow mod authors to know if their account was compromised as they would get an email if a file was uploaded to Modrinth for their account but the modder did not do that themselves. Then the can go ahead and take action to remove the file and secure their account. Will help greatly reduce the time between malicious files going out and discovery of the issue

**Comment by Lylythii** [comment]
Thought of an idea that'd complement something like this. Should authors want it.

Ways for an author to confirm publicly listing new mod uploads.
So even if a malicious actor does upload a mod file on your behalf, & it slips through all checks/manual reviews. The author would need to confirm that "Yes this is me uploading the new file.", before it's publicly listed & downloadable. Should the author say "I didn't do that." the file would be rejected.

Presumably by sending a confirmation request to a mobile device or other application. Or as you suggested, Email.

**Comment by AnOpenSauceDev** [comment]
This should also happen for mod removal/replace operations, because that would be a much harder thing to identify without notifications.

**Comment by rotgruengelb** [comment]
Yes, this should be at least an option for people to enable in their Settings.

And off course if is a toggleable Feature send a notification via E-Mail if the Feature is toggled off.
